front/lib/models/assistant/conversation.ts

```diff
@@ -718,7 +718,10 @@ export class Mention extends WorkspaceAwareModel<Mention> {
   declare updatedAt: CreationOptional<Date>;
 
   declare messageId: ForeignKey<Message["id"]>;
+
+  // a Mention is either an agent mention xor a user mention
   declare agentConfigurationId: string | null; // Not a relation as global agents are not in the DB
+  declare userId: ForeignKey<UserModel["id"]> | null;
```

Comment:
we add the new link between a user and a mention so we can support user mentions